docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
       - CONTRACT_FUNGIBLE_TOKEN=0xee82856bf20e2aa6
       - CONTRACT_FLOW_TOKEN=0x0ae53cb6e3f42a79
       - CONTRACT_FUSD=0xf8d6e0586b0a20c7
+      - CONTRACT_FCL_CRYPTO=0xf8d6e0586b0a20c7
```

Comment:
Not sure what network this address should reference (currently set to emulator)

Reply:
I think emulator is right, since the dev wallet is really only designed for use with the emulator.